packages/side-nav/src/SideNav.tsx

```diff
@@ -5,6 +5,8 @@ import { ulStyleOverrides } from './styles';
 
 const navStyles = css`
   width: 200px;
+  overflow-y: scroll;
```

Comment:
> Use `auto` instead

Good call, I missed this because macOS hides scrollbars when not in use.

>  I’m not convinced this should ever scroll independent of the page

I was careful to preserve existing behavior, ex: [Testing Cluster on QA](https://cloud-qa.mongodb.com/v2/5e18d3e0a7cc6835f8f7d364#clusters)

> Who suggested this change?

Cory Mintz (VP of Atlas Engineering) brought this up this morning (Feb 14). After he did, several other teams throughout Cloud reached out about the same, mostly people who use Atlas (which has a gigantic side nav).

**However**, I totally agree with you, and think it definitely reasonable to say that `<SideNav>` should be agnostic in terms of scrolling and bottom padding related to scrolling, should anyone mention it outside of Cloud. Users can either just use a container, or style the `<SideNav>` themselves if they use Emotion (I'm merging a PR right now to do that).

Reply:
An update – Harry, Nelle, and I are talking about the right solution for this. The main factors are whether this is simply a product of the transitional phase, or if this will be an issue in products after the transition. If it's the former, I'm likely to suggest adding these changes as overrides within MMS itself.

I appreciate moving the scrolling behavior to MMS, as that specific issue seems specific to there.

As an FYI, you should be able to pass a non-emotion className as a string to our components, and have that style applied as well. `cx()` should handle that case by doing old-school string concatenation between the string className and the emotion className.